### DISCUSSION TITLE
How can I chain commands in the CLI image?

### DISCUSSION THREAD

**Discussion by Luc45** [docker-library/wordpress#810] [General]
I want to run multiple commands at once for performance reasons, eg:

```
docker-compose run --rm tests-cli "wp --version && wp --version && wp --version"
```

How can I chain commands, instead of the entrypoint considering each command as an argument to `wp`?

**Comment by tianon** [comment]
You'll want to run something like `sh -c` as your command (essentially, you'll need to run some kind of shell to get shell-like behavior :+1:):

```console
$ docker-compose run --rm tests-cli sh -c "wp --version && wp --version && wp --version"
```

**Reply by Luc45** [reply]
Thank you! It works.
